.github/workflows/bench.yml

```diff
@@ -36,6 +36,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
```

Comment:
We have no submodules nor plans to have hem so i think we could remove it from all workflows

Reply:
Well, I picked the submodule part from some workflow checkout & since trying to make everything seamless - included the submodule directive. Ok removing it.